spec/active_record/events_spec.rb

```diff
@@ -116,5 +116,22 @@
       task.update_columns(expired_at: nil)
       expect(Task.not_expired).to include(task)
     end
+
+    describe 'for date fields' do
+      it 'consider today\'s event over' do
+        task.update_columns(notified_on: Date.today)
+        expect(task.notified?).to eq(true)
```

Comment:
Could you also check if the scope returns the record? Similar to: https://github.com/pienkowb/active_record-events/blob/d727d184ff9eec0a28f92a24ef4157ebd2724695/spec/active_record/events_spec.rb#L103

Reply:
nice idea. done.